docker-compose.yml

```diff
@@ -1,15 +1,25 @@
 version: "3"
 
 services:
+  bitcoinz-insight-patched:
+    container_name: bitcoinz-insight-patched
```

Comment:
I'm not a fan of containers with a name. Since you're `link`ing this below, this isn't necessary.

Reply:
Link is required so that the server name is resolvable by DNS. Do you have a better suggestion?

Reply:
Link is required, but `container_name` shouldn't be.